plugins/callback/log_plays.py

```diff
@@ -47,13 +47,13 @@ class CallbackModule(CallbackBase):
     """
     logs playbook results, per host, in /var/log/ansible/hosts
     """
-    CALLBACK_VERSION = 2.0
+    CALLBACK_VERSION = 2.1
```

Comment:
Why do you do that? Is that a valid value? I can only see 1.0 and 2.0 in use anywhere.

Reply:
Thanks for your review @felixfontein , and sorry about those mistakes.
Fixed.